scripts/benchmarks/configuration.sh

```diff
@@ -30,8 +30,8 @@ export ZEITGEIST_PALLETS=(
     zrml_authorized zrml_court zrml_global_disputes zrml_liquidity_mining zrml_neo_swaps \
     zrml_orderbook zrml_parimutuel zrml_prediction_markets zrml_swaps zrml_styx \
 )
-export ZEITGEIST_PALLETS_RUNS="${ZEITGEIST_PALLETS_RUNS:-1000}"
-export ZEITGEIST_PALLETS_STEPS="${ZEITGEIST_PALLETS_STEPS:-10}"
+export ZEITGEIST_PALLETS_RUNS="${ZEITGEIST_PALLETS_RUNS:-20}"
+export ZEITGEIST_PALLETS_STEPS="${ZEITGEIST_PALLETS_STEPS:-50}"
```

Comment:
I always see the default to be runs: 50 and steps: 20. Seems like you could have meant that.

Reply:
Please elaborate. All other pallets within our repo use `RUNS=20` and `STEPs=50` and Substrate pallets in general also use this configuration, as can be seen at https://github.com/paritytech/polkadot-sdk/blob/a42a47f81d91ccf2f885ff01749185cd4c793278/substrate/frame/balances/src/weights.rs#L30-L31